src/main/webapp/jsp/module/purap/PaymentRequest.jsp

```diff
@@ -1,123 +1,125 @@
-                displayPaymentRequestInvoiceInfoFields="true"/>
-
-        <c:if test="${taxInfoViewable || taxAreaEditable}">
-            <purap:paymentRequestTaxInfo
-                    documentAttributes="${DataDictionary.PaymentRequestDocument.attributes}"/>
-        </c:if>
-
-        <purap:paymentRequestProcessItems
-                documentAttributes="${DataDictionary.PaymentRequestDocument.attributes}"
-                itemAttributes="${DataDictionary.PaymentRequestItem.attributes}"
-                accountingLineAttributes="${DataDictionary.PaymentRequestAccount.attributes}"/>
-
-        <purap:summaryaccounts
-                itemAttributes="${DataDictionary.PaymentRequestItem.attributes}"
-                documentAttributes="${DataDictionary.SourceAccountingLine.attributes}"/>
-
-        <purap:relatedDocuments />
-
-        <purap:paymentHistory />
-
-        <purap:preqWireTransfer/>
-        <purap:preqForeignDraft/>
-
-        <gl:generalLedgerPendingEntries/>
-
-        <kul:notes
-                attachmentTypesValuesFinder="${DataDictionary.PaymentRequestDocument.attachmentTypesValuesFinder}"
-                defaultOpen="${KualiForm.document.openAttachmentTab}"/>
-
-        <kul:adHocRecipients/>
-
-        <kul:routeLog/>
-
-        <kul:superUserActions/>
-    </c:if>
-
-    <c:set var="extraButtons" value="${KualiForm.extraButtons}"/>
-    <sys:documentControls
-            transactionalDocument="true"
-            extraButtons="${extraButtons}"
-            suppressRoutingControls="${KualiForm.editingMode['displayInitTab']}"
-            tabindex="${globalButtonTabIndex}"/>
-    <kul:modernLookupSupport />
-</kul:documentPage>
+<%--
+
+    The Kuali Financial System, a comprehensive financial management system for higher education.
+
+    Copyright 2005-2023 Kuali, Inc.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+--%>
+<!-- KualiCo 2023-04-19 base code version of the JSP with Cornell customizations applied. -->
+
+<%@ include file="/jsp/sys/kfsTldHeader.jsp" %>
+
+<kul:documentPage showDocumentInfo="true"
+                  documentTypeName="PaymentRequestDocument"
+                  htmlFormAction="purapPaymentRequest" renderMultipart="true"
+                  showTabButtons="true">
+
+    <!-- Cornell Customization -->
```

Comment:
There are some other files in this PR that are using the HTML-based custom comments instead of the JSP/TAG comment format. Should those custom comments be updated to the JSP/TAG convention as well?

Reply:
Adjustments made.